src/main/resources/flows-config.yaml

```diff
@@ -296,31 +296,100 @@ flow:
       - name: householdHomeExpenses
   householdHomeExpenses:
     nextScreens:
-      - name: householdEmployerName
-  expensesHome:
-    nextScreens:
+      - name: expensesRent
+        condition: ExpensesIncludeRent
```

Comment:
@sree-cfa @lkemperman-cfa - I don't love this approach. Is there a better way?

Reply:
Yeah, I dont think there's a better way atm. The only thing I can think of is extracting some of the duplicate code into a super conditional class so that each condition just has to pass the corresponding value

Reply:
Ex. 
```
return super.run(submission, "Rent")
```